models/frax_finance/bnb/frax_finance_bnb_bribes.sql

```diff
@@ -0,0 +1,462 @@
+        sum(amount) as bribe
+    from
+        bribe_base a
+        left join pairs c on a.contract_address = c.contract_address
+    group by
+        1,
+        2
+),
+bribes_received as (
+    select
+        block_time as collection_time,
+        tx_hash,
+        a.contract_address as bribe_address,
+        c.contract_address,
+        concat('0x', substring(topic3, 27, 42)) as reward_token_collected,
+        round(bytea2numeric_v3(substr(data, 3)) / pow(10, 18)) as reward_amount,
+        case
+            when substring(topic3, 27, 42) = 'e48a3d7d0bc88d552f730b62c006bc925eadb9ee' then round(
+                bytea2numeric_v3(substr(data, 3)) / pow(10, 18) * b.price
+            )
+        end as reward_amount_usd
+    from
+        { { source('bnb', 'logs') } } a
+        left join (
+            select
+                *
+            from
+                prices
+            where
+                contract_address = '0x3432b6a60d23ca0dfca7761b7ab56459d9c964d0'
+        ) b on date_trunc('day', a.block_time) = b.day
+        left join all_addresses c on a.contract_address = c.bribe_address
+    where
+        topic1 = '0x540798df468d7b23d11f156fdb954cb19ad414d150722a7b6d55ba369dea792e'
+        and topic2 = '0x0000000000000000000000008811da0385ccf1848b21475a42ea4d07fc5d964a'
+        and a.contract_address in (
+            select
+                bribe_address
+            from
+                all_addresses
+            where
+                c_type = 'bribe contract'
+        )
+        and block_time > cast('2023-01-02' as date)
+),
+weekly_rewards as (
+    select
+        collection_time,
+        bribe_address,
+        contract_address,
+        sum(reward_amount_usd) as FXS_collected_usd,
+        sum(reward_amount) as FXS_collected
+    from
+        bribes_received
+    group by
+        1,
+        2,
+        3
+),
+fees_collected as (
+    select
+        block_time as collection_time,
+        tx_hash,
+        a.contract_address as bribe_address,
+        c.contract_address,
+        case
+            when concat('0x', substring(topic3, 27, 42)) = c.token0 then round(
+                bytea2numeric_v3(substr(data, 3)) / pow(10, 18),
+                4
+            )
+        end as fee_reward_token0,
+        case
+            when concat('0x', substring(topic3, 27, 42)) = c.token1 then round(
+                bytea2numeric_v3(substr(data, 3)) / pow(10, 18),
+                4
+            )
+        end as fee_reward_token1
+    from
+        bnb.logs  { { source('bnb', 'logs') } } a
+        left join all_addresses c on a.contract_address = c.bribe_address
+    where
+        topic1 = '0x540798df468d7b23d11f156fdb954cb19ad414d150722a7b6d55ba369dea792e'
+        and topic2 = '0x0000000000000000000000008811da0385ccf1848b21475a42ea4d07fc5d964a'
+        and a.contract_address in (
+            select
+                bribe_address
+            from
+                all_addresses
+            where
```

Comment:
Hi @Vahid-flipside , is it possible to modify this view into an incremental view by adding incremental filters?

Reply:
@Hosuke You mean for the materialization of the model to be incremental table as opposed to a view?

Reply:
BTW, this view is not scanning a large number of rows in Logs table and its size is very small.

Reply:
Is `inner jon` possible to use in this case?